cartridges/app_storefront_pdp_extend/cartridge/templates/default/product/components/notifySubscription.isml

```diff
@@ -0,0 +1,46 @@
+<iscache status="off"/>
```

Comment:
Please use middlewares to set cache status. Also the template is locally included hence will impact the caching of the parent template.